build error

**build error if disable NNG_SUPP_HTTP**
`nng-2.0.0-alpha.5/src/supplemental/http/http_public.c:292:24: error: ‘res’ undeclared (first use in this function)
  292 |         NNI_ARG_UNUSED(res);
      |    `


** Environment Details **
 - NNG version: 2.0.0-alpha.5
 - Operating system and version: Linux
 - Compiler and language used: GCC
 - Shared or static library: both

--- a/src/supplemental/http/http_public.c
+++ b/src/supplemental/http/http_public.c
@@ -289,7 +289,6 @@ nng_http_read_response(nng_http *conn, nng_aio *aio)
 	nni_http_read_res(conn, aio);
 #else
 	NNI_ARG_UNUSED(conn);
-	NNI_ARG_UNUSED(res);
 	nni_aio_finish_error(aio, NNG_ENOTSUP);
 #endif
 }

